packages/analytics-react-native/test/react-native-client.test.ts

```diff
@@ -572,26 +572,26 @@ describe('react-native-client', () => {
         // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
         const events = send.mock.calls.flatMap((call) => call[1].events as Event[]);
         expect(events.length).toEqual(18);
-        events.forEach((event, i) => expect(event.event_id).toEqual(i));
+        events.forEach((event, i) => expect(event.event_id).toEqual(i + 1));
 
         expect(events[0].event_type).toEqual('session_end');
         expect(events[0].session_id).toEqual(500);
         expect(events[0].time).toEqual(851);
 
         expect(events[1].event_type).toEqual('session_start');
-        expect(events[1].session_id).toEqual(950);
-        expect(events[1].time).toEqual(950);
+        expect(events[1].session_id).toEqual(1000);
```

Comment:
Is this because the app starts in the background? I would expect the session to start on init.

Reply:
Yes, in this test the app starts in the background.
Case A: the app starts at time 950, does something (not tracked to Amplitude), closes
Case B: the app starts at time 950, track an event  at time 1000, closes.

Option 1:
Case A: session_start(950), session_end(950) - here we have empty session
Case B: session_start(950), event(1000), session_end(1000)

Option 2:
Case A: -
Case B: session_start(950), event(1000), session_end(1000)

Option 3:
Case A: -
Case B: session_start(1000), event(1000), session_end(1000)

Now I do Option 3. Do you expect Option 1?

Reply:
Thanks, in this case I would expect `Option 1`. This will become more important to track "empty" sessions since we are being to add session replay functionality that will depend on session start/end regardless of other Amplitude track()ing.

Reply:
Reverted to `Option 1`